utils/qemu/patches/0010-mips-softfloat.patch

```diff
@@ -0,0 +1,33 @@
+This patch is needed due to our mips64 build uses softfloat. Qemu will not
+build without this patch.
+
+--- a/tests/fp/fp-bench.c
++++ b/tests/fp/fp-bench.c
```

Comment:
Can we patch out `subdir('tests` in meson.build?

Reply:
I guess so.

Reply:
Done.